tests/unit/Rules.spec.ts

```diff
@@ -0,0 +1,185 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createPinia, setActivePinia } from 'pinia'
+import { useStore } from '@/store/store'
+import { mount } from '@vue/test-utils'
+import Rules from '@/components/SpecialResolution/Rules.vue'
+import UploadRules from '@/components/SpecialResolution/UploadRules.vue'
+
+Vue.use(Vuetify)
+const vuetify = new Vuetify({})
+
+setActivePinia(createPinia())
+const store = useStore()
+
+describe('Rules', () => {
+  let wrapper: any
+  beforeEach(async () => {
+    store.stateModel.rules = {}
+    store.stateModel.entitySnapshot = null
+    wrapper = mount(Rules, { vuetify })
+  })
+
+  it('renders the component', () => {
+    expect(wrapper.exists()).toBe(true)
+  })
+
+  it('saveRules - valid - rules in resolution', async () => {
+    await wrapper.find('#btn-change-rules').trigger('click')
+    await wrapper.find('#btn-describe-rules').trigger('click')
+    await wrapper.find('#chk-rules-describe').trigger('click')
+    await wrapper.find('#btn-rules-done').trigger('click')
```

Comment:
Clicking all 4 controls in a row works? Sweet. (We struggled with this previously.)